ios/CredentialsManagerBridge.swift

```diff
@@ -96,7 +96,7 @@ extension Credentials {
             CredentialsManagerBridge.tokenTypeKey: self.tokenType,
             CredentialsManagerBridge.idTokenKey: self.idToken,
             CredentialsManagerBridge.refreshTokenKey: self.refreshToken as Any,
-            CredentialsManagerBridge.expiresInKey: self.expiresIn,
+            CredentialsManagerBridge.expiresInKey: self.expiresIn.timeIntervalSince1970,
```

Comment:
```suggestion
            CredentialsManagerBridge.expiresInKey: floor(self.expiresIn.timeIntervalSince1970),
```

Can we floor this value so that it is returned without decimal points?